content/en/docs/hertz/tutorials/service-governance/open-telemetry.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "OpenTelemetry"
+date: 2022-09-13
+weight: 4
+description: >
+---
+
+[OpenTelemetry](https://opentelemetry.io/) is an open source observability framework
+for [CNCF](https://www.cncf.io/), which is a set of tools, APIs, and SDKs, and it enables IT teams to detect, generate,
```

Comment:
OpenTelemetry is an open source observability framework from CNCF that consist of a series of tools, APIs and SDKS.

Reply:
done